.golangci.yaml

```diff
@@ -47,7 +54,13 @@ linters:
                 # This will force omitzero on required struct fields.
                 # This means they can be omitted correctly and prevents the need for pointers to structs.
                 policy: SuggestFix
-            uniqueMarkers:
+            namingconventions:
+              conventions:
```

Comment:
I wonder if it would make sense to add one for https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#use-resource-name-rather-than-kind-in-object-references ?

I think this could have a violation matcher that catches `Kind` naming and suggest a fix that it should be `Resource` instead?

Reply:
Good shout